app/views/spree/admin/shared/_order_tabs.html.haml

```diff
@@ -46,18 +49,27 @@
 
   %nav.menu
     %ul
-      %li{ class: "#{'active' if true}" }
+      - order_details_classes = "active" if current == "Order Details"
```

Comment:
If we ever change this code again, using the same pattern in six places justifies the use of a helper. It's also a good place for a comment explaining why we do this:
```rb
# Due to a bug in HAML, we can't inline CSS classes and need to assign them via a variable or helper.
# See: https://github.com/openfoodfoundation/openfoodnetwork/pull/2955
def tab_active_class(current, tab_entry)
  if current == tab_entry
    "active"
  else
    ""
  end
end
```
This is in test ready already. So I wouldn't worry about it. Just an idea for future work.

Reply:
good catch @mkllnk !